Rakefile

```diff
@@ -19,3 +19,21 @@ RuboCop::RakeTask.new do |task|
   task.requires << 'rubocop-performance'
   task.requires << 'rubocop-rspec'
 end
+
+require 'rails'
+require 'erb'
+
+task :environment do
```

Comment:
thanks to https://snippets.aktagon.com/snippets/257-how-to-use-activerecord-without-rails